frontend/src/Table.js

```diff
@@ -98,4 +98,41 @@ const Table = (props) => {
   );
 };
 
-export default Table;
\ No newline at end of file
+// TODO's: (i) finish this implementation: "x"s -> checkmarks?, (ii) matrix widget too/instead? (header x side header)
+const ComparisonTable = (props) => {
```

Comment:
This is basically a simplified version of your other Table widget. It may be possible to combine these together, or maybe we can keep them separate. im not sure yet